packages/x-adapter-platform/src/types/requests/request.types.ts

```diff
@@ -0,0 +1,56 @@
+/**
```

Comment:
Export just the needed ones:

  PlatformQueryableRequest, PlatformExtraParamsRequest